server/server_controller.py

```diff
@@ -20,6 +20,7 @@ def create_server(self, name, host, port):
             'connection': None,
             'address': None
         }
+        print("Server named %s " %name + "created in host %s " %host + "and port %s." %port)
```

Comment:
print("Server named %s was created with host %s and port %s." % (name, host, port))